Framework/Core/src/DataProcessingDevice.cxx

```diff
@@ -95,6 +98,7 @@ DataProcessingDevice::HandleData(FairMQParts &iParts, int /*index*/) {
   // does not need to know about the whole class state, but I can 
   // fine grain control what is exposed at each state.
   auto &metricsService = mServiceRegistry.get<MetricsService>();
+  auto& monitoringService = mServiceRegistry.get<Monitoring>();
```

Comment:
For the record, as we discussed, I think we should replace completely the `MetricsService` with` Monitoring`.